meteor-app/imports/ui/assessment/assessment-job-card.js

```diff
@@ -1,58 +1,106 @@
+import { Meteor } from 'meteor/meteor'
+import { Component } from 'react'
 import { Card, Button, Container, List } from 'semantic-ui-react'
 import "/imports/ui/layouts/assessment.css"
-import CONSTANTS from '/imports/api/constants'
-
+import { JOB_STATUS, JOB_STATUS_BUTTON } from '/imports/api/constants'
+import printJobCart from '/imports/ui/assessment/assessment-print-job'
 
 class JobCard extends Component {
 
+  updateButton = () => {
+    const jobId = this.props.currentJob._id
+    const status = this.props.currentJob.status
+    const updatedStatus = status+1
 
-    render() {
-        const { status, bikeDetails, services, pickupDate, totalCost } = this.props.currentJob
-        const make = bikeDetails.make
-        const model = bikeDetails.model
-        const color = bikeDetails.color
-        const serviceLevel = services.baseService
-        const pickUp = pickupDate.toLocaleDateString()
-        return (
-            <Card>
-                <Container className="job-card-container" >
-                    <Card.Header style={{ textAlign: "Center", fontSize: "1.5em", margin: "10px" }} >
-                        {`Job Status: ${CONSTANTS.ORDER_STATUS_READABLE[status]}`}
-                    </Card.Header>
-                    <Card.Content style={{ textAlign: "Center", fontSize: "1em" }}>
-                        <List>
-                            <List.Item>{`Make: ${make}`}</List.Item>
-                            <List.Item>{`Model: ${model}`}</List.Item>
-                            <List.Item>{`Color: ${color}`}</List.Item>
-                            <List.Item>{`Service Level: ${serviceLevel}`}</List.Item>
-                            <List.Item>{`Pick-Up Date: ${pickUp}`}</List.Item>
-                            <List.Item>{`Total Price: $${totalCost/100}`}</List.Item>
-                        </List>
-                    </Card.Content>
-                    <Container style={{ textAlign: "Center" }}>
-                        <Button.Group style={{  width: "80%" }} vertical>
-                            <Button className="positive ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Start Job</h2>
-                            </Button>
-                            <Button className="positive ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Print Job Card</h2>
-                            </Button>
-                            <Button className="negative ui button"
-                            style={{ marginTop: '5px', marginBottom: '5px' }} >
-                                <h2>Cancel Job</h2>
-                            </Button>
-                        </Button.Group>
-                    </Container>
-                </Container>
-            </Card>
+    try {
+      if (status >= 5) return
+      Meteor.call('assessment.updateJobStatus', jobId, updatedStatus)
```

Comment:
You should not call Meteor methods from a React component, because storybook won't work. You should pass in a function from the container component, which it can call.